tests/test_keyring_storage.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for oauth2client.keyring_storage tests.
+
+Unit tests for oauth2client.keyring_storage.
+"""
+
+import datetime
+import keyring
+import thread
```

Comment:
Aw man! I was sleepy and figured it would work fine on Travis! Boo.

It doesn't look like `thread` is in `six.moves` so maybe I'll just leave it out. The only use is in `test_constructor`:

```python
self.assertTrue(isinstance(store._lock, thread.LockType))
```